plugins/BEdita/Core/src/Model/Behavior/DataCleanupBehavior.php

```diff
@@ -51,6 +54,9 @@ class DataCleanupBehavior extends Behavior
     public function beforeMarshal(Event $event, \ArrayObject $data)
     {
         $config = $this->getConfig();
+        $objectType = Inflector::underscore($this->_table->getAlias());
```

Comment:
Maybe it's better take the "real" object type `name` since the alias could is different for example for relation.

```php
if ($this->_table->behaviors()->has('ObjectType')) {
     $defaults = Configure::read(sprintf('DefaultValues.%s', $this->_table->objectType()->get('name')), []);
     $config['fields'] = array_merge($config['fields'], $defaults);
}
```

But since the behavior could be used with any table (not only tables that represents object type) we could also generalize as

```php
$key = Inflector::underscore($this->_table->getAlias());

if ($this->_table->behaviors()->has('ObjectType')) {
     $key = $this->_table->objectType()->get('name');
}

$defaults = Configure::read(sprintf('DefaultValues.%s', $key), []);
$config['fields'] = array_merge($config['fields'], $defaults);
```